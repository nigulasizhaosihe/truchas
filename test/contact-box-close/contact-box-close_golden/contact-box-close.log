
================================================================================

PROGRAM INFORMATION

   code:                Truchas 21.06-10-geff71803-dirty
   build architecture:  x86_64
   build date/time:     2021-07-29 16:44:12
   build flags:          -fimplicit-none -ffree-line-length-none -O3 -DNDEBUG
   build host:          theseus
   run architecture:    Linux theseus 5.10.52-1-lts #1 SMP Tue, 20 Jul
   run host:            theseus
   run date/time:       30 Jul 21 15:16:24
   processors:          1

================================================================================

INPUT

Opening input file "contact-box-close.inp"
Reading FUNCTION namelists ...
  read namelist "applied temperature profile"
Reading VFUNCTION namelists ...
  none found
Reading TOOLPATH namelists ...
  none found
Reading PHYSICS namelist ...
Reading MATERIAL namelists ...
  read namelist "aluminum"
Reading PHASE namelists ...
  none found
Reading PHASE_CHANGE namelists ...
  none found
Reading OUTPUTS namelist ...
Reading MESH namelist ...
Reading BODY namelists (first pass) ...
  read 1 BODY namelist
Reading BODY namelists (second pass) ...
  read 1 BODY namelist
Reading NUMERICS namelist ...

Reading SOLID_MECHANICS namelist ...

Reading SM_BC namelists ...
Reading DIFFUSION_SOLVER namelist ...
  using default STEPPING_METHOD value: "Adaptive BDF2"
  using default MAX_NLK_ITR value: 5
  using default MAX_STEP_TRIES value: 10
  using default MAX_NLK_VEC value: MAX_NLK_ITR - 1
  using default NLK_VEC_TOL value: 1.00E-03
  WARNING: using a pure relative error norm; temp must be bounded away from 0
  WARNING: using a pure relative error norm; enthalpy must be bounded away from 0
  using default PC_AMG_CYCLES value: 2
Reading THERMAL_BC namelists ...
  read namelist "ends"
  read namelist "lateral sides"
  read namelist "gap interface"
Reading THERMAL_SOURCE namelists ...
  none found
Reading SPECIES_BC namelists ...
  none found
Reading DS_SOURCE namelists ...
  none found
Reading ENCLOSURE_RADIATION namelists ...
  none found
Reading PROBE namelists ...
  none found
Input file "contact-box-close.inp" closed

================================================================================

INITIALIZATION

Initializing mesh "MAIN" ...
  reading ExodusII mesh file "./rotated-cube.exo"

Computing initial volume fractions ...
  using SM_BC[left/right/front/bottom/top ends]
  using SM_BC[gap interface]
SM BC faces:    112
SM BC nodes/links:    126    25
SM-D1 nodes:     46
SM-D2 nodes:     26
SM-D3 nodes:      4
SM-C1 nodes:     24
SM-C1D1 nodes:     22
SM-C1D2 nodes:      4
Nodes with requested BCs:    126    Nodes with applied BCs:    126
SM: Computing initial state...


Initializing diffusion solver ...
  generating external source for "temperature" equation
  generating "scalar_cell_func1" thermal source
    none specified
  generating "interface-htc" thermal interface condition
    using THERMAL_BC[gap interface]
  generating "gap-radiation" thermal interface condition
    none specified
  generating "flux" thermal boundary condition
    using THERMAL_BC[lateral sides]
  generating "htc" thermal boundary condition
    none specified
  generating "radiation" thermal boundary condition
    none specified
  generating "temperature" thermal boundary condition
    using THERMAL_BC[ends]
  diffusion solver initialized

Computing consistent initial state for HT/SD solver ...
  computing consistent face temperatures and radiosities ...

Computing consistent initial state derivative for HT/SD solver ...
  computing consistent face temperatures and radiosities ...
  ||Rface(2)||= 5.54E-05, ||Rface(0)||= 5.27E-02, ||Î”Tface||_max= 3.17E-05

Initializing solution probes ...

================================================================================

EXECUTION

          1: t =   0.00000E+00, dt(initial) =   1.00000E+00

DS: dt=1.000E+00, NFUN:NPC=0000002:00001, NNR:NNF:NSR=0000:0000:0000

          2: t =   1.00000E+00, dt(diffusion solver) =   1.00000E+00

DS: dt=1.000E+00, NFUN:NPC=0000005:00001, NNR:NNF:NSR=0000:0000:0000

          3: t =   2.00000E+00, dt(diffusion solver) =   1.00000E+00

DS: dt=1.000E+00, NFUN:NPC=0000008:00001, NNR:NNF:NSR=0000:0000:0000

          4: t =   3.00000E+00, dt(diffusion solver) =   1.00000E+00

DS: dt=1.000E+00, NFUN:NPC=0000011:00001, NNR:NNF:NSR=0000:0000:0000

          5: t =   4.00000E+00, dt(diffusion solver) =   2.67971E+00

DS: dt=2.680E+00, NFUN:NPC=0000014:00001, NNR:NNF:NSR=0000:0000:0000

          6: t =   6.67971E+00, dt(diffusion solver) =   2.48307E+00

DS: dt=2.483E+00, NFUN:NPC=0000017:00001, NNR:NNF:NSR=0000:0000:0000

          7: t =   9.16278E+00, dt(diffusion solver) =   4.87834E+00

DS: dt=4.878E+00, NFUN:NPC=0000020:00002, NNR:NNF:NSR=0000:0000:0000

          8: t =   1.40411E+01, dt(diffusion solver) =   3.76068E+00

DS: dt=3.761E+00, NFUN:NPC=0000023:00002, NNR:NNF:NSR=0000:0000:0000

          9: t =   1.78018E+01, dt(diffusion solver) =   3.82762E+00

DS: dt=3.828E+00, NFUN:NPC=0000026:00002, NNR:NNF:NSR=0000:0000:0000

         10: t =   2.16294E+01, dt(diffusion solver) =   4.00100E+00

DS: dt=4.001E+00, NFUN:NPC=0000029:00002, NNR:NNF:NSR=0000:0000:0000

         11: t =   2.56304E+01, dt(diffusion solver) =   3.81369E+00

DS: dt=3.814E+00, NFUN:NPC=0000032:00002, NNR:NNF:NSR=0000:0000:0000

         12: t =   2.94441E+01, dt(diffusion solver) =   3.97628E+00

DS: dt=3.976E+00, NFUN:NPC=0000035:00002, NNR:NNF:NSR=0000:0000:0000

         13: t =   3.34204E+01, dt(time) =   3.73795E+00

DS: dt=3.738E+00, NFUN:NPC=0000038:00002, NNR:NNF:NSR=0000:0000:0000

         14: t =   3.71583E+01, dt(time) =   3.51391E+00

DS: dt=3.514E+00, NFUN:NPC=0000041:00002, NNR:NNF:NSR=0000:0000:0000

         15: t =   4.06722E+01, dt(time) =   3.30329E+00

DS: dt=3.303E+00, NFUN:NPC=0000044:00002, NNR:NNF:NSR=0000:0000:0000

         16: t =   4.39755E+01, dt(time) =   3.10530E+00

DS: dt=3.105E+00, NFUN:NPC=0000047:00002, NNR:NNF:NSR=0000:0000:0000

         17: t =   4.70808E+01, dt(time) =   2.91918E+00

DS: dt=2.919E+00, NFUN:NPC=0000050:00002, NNR:NNF:NSR=0000:0000:0000

         18: t =   5.00000E+01, dt(time) =   1.84010E+00

DS: dt=1.840E+00, NFUN:NPC=0000053:00002, NNR:NNF:NSR=0000:0000:0000

         19: t =   5.18401E+01, dt(time) =   1.15990E+00

DS: dt=1.160E+00, NFUN:NPC=0000056:00002, NNR:NNF:NSR=0000:0000:0000

         20: t =   5.30000E+01, dt(diffusion solver) =   4.63961E+00

DS: dt=4.640E+00, NFUN:NPC=0000059:00002, NNR:NNF:NSR=0000:0000:0000

         21: t =   5.76396E+01, dt(diffusion solver) =   3.62487E+00

DS: dt=3.625E+00, NFUN:NPC=0000062:00002, NNR:NNF:NSR=0000:0000:0000

         22: t =   6.12645E+01, dt(diffusion solver) =   4.01495E+00

DS: dt=4.015E+00, NFUN:NPC=0000065:00002, NNR:NNF:NSR=0000:0000:0000

         23: t =   6.52794E+01, dt(diffusion solver) =   4.02828E+00

DS: dt=4.028E+00, NFUN:NPC=0000068:00002, NNR:NNF:NSR=0000:0000:0000

         24: t =   6.93077E+01, dt(diffusion solver) =   3.92933E+00

DS: dt=3.929E+00, NFUN:NPC=0000071:00002, NNR:NNF:NSR=0000:0000:0000

         25: t =   7.32370E+01, dt(diffusion solver) =   4.00323E+00

DS: dt=4.003E+00, NFUN:NPC=0000074:00002, NNR:NNF:NSR=0000:0000:0000

         26: t =   7.72403E+01, dt(diffusion solver) =   3.96598E+00

DS: dt=3.966E+00, NFUN:NPC=0000077:00002, NNR:NNF:NSR=0000:0000:0000

         27: t =   8.12063E+01, dt(diffusion solver) =   4.00371E+00

DS: dt=4.004E+00, NFUN:NPC=0000080:00002, NNR:NNF:NSR=0000:0000:0000

         28: t =   8.52100E+01, dt(diffusion solver) =   3.99528E+00

DS: dt=3.995E+00, NFUN:NPC=0000083:00002, NNR:NNF:NSR=0000:0000:0000

         29: t =   8.92053E+01, dt(diffusion solver) =   4.00254E+00

DS: dt=4.003E+00, NFUN:NPC=0000086:00002, NNR:NNF:NSR=0000:0000:0000

         30: t =   9.32078E+01, dt(diffusion solver) =   4.00237E+00

DS: dt=4.002E+00, NFUN:NPC=0000089:00002, NNR:NNF:NSR=0000:0000:0000

         31: t =   9.72102E+01, dt(diffusion solver) =   4.00292E+00

DS: dt=4.003E+00, NFUN:NPC=0000092:00002, NNR:NNF:NSR=0000:0000:0000

         32: t =   1.01213E+02, dt(diffusion solver) =   4.84526E+00
Diffusion solver step failed; retrying with reduced step size

DS: dt=2.423E+00, NFUN:NPC=0000099:00002, NNR:NNF:NSR=0000:0000:0001

         33: t =   1.03636E+02, dt(diffusion solver) =   1.45661E+00

DS: dt=1.457E+00, NFUN:NPC=0000103:00002, NNR:NNF:NSR=0000:0000:0001

         34: t =   1.05092E+02, dt(diffusion solver) =   2.47921E+00

DS: dt=2.479E+00, NFUN:NPC=0000106:00002, NNR:NNF:NSR=0000:0000:0001

         35: t =   1.07572E+02, dt(diffusion solver) =   2.14246E+00

DS: dt=2.142E+00, NFUN:NPC=0000109:00002, NNR:NNF:NSR=0000:0000:0001

         36: t =   1.09714E+02, dt(diffusion solver) =   3.95678E+00

DS: dt=3.957E+00, NFUN:NPC=0000112:00002, NNR:NNF:NSR=0000:0000:0001

         37: t =   1.13671E+02, dt(maximum) =   5.00000E+00

DS: dt=5.000E+00, NFUN:NPC=0000114:00002, NNR:NNF:NSR=0000:0000:0001

         38: t =   1.18671E+02, dt(maximum) =   5.00000E+00

DS: dt=5.000E+00, NFUN:NPC=0000116:00002, NNR:NNF:NSR=0000:0000:0001

         39: t =   1.23671E+02, dt(maximum) =   5.00000E+00

DS: dt=5.000E+00, NFUN:NPC=0000118:00002, NNR:NNF:NSR=0000:0000:0001

         40: t =   1.28671E+02, dt(time) =   4.73763E+00

DS: dt=4.738E+00, NFUN:NPC=0000120:00002, NNR:NNF:NSR=0000:0000:0001

         41: t =   1.33408E+02, dt(time) =   4.48903E+00

DS: dt=4.489E+00, NFUN:NPC=0000121:00002, NNR:NNF:NSR=0000:0000:0001

         42: t =   1.37897E+02, dt(time) =   4.25348E+00

DS: dt=4.253E+00, NFUN:NPC=0000123:00002, NNR:NNF:NSR=0000:0000:0001

         43: t =   1.42151E+02, dt(time) =   4.03028E+00

DS: dt=4.030E+00, NFUN:NPC=0000125:00002, NNR:NNF:NSR=0000:0000:0001

         44: t =   1.46181E+02, dt(time) =   3.81880E+00

DS: dt=3.819E+00, NFUN:NPC=0000126:00002, NNR:NNF:NSR=0000:0000:0001

================================================================================

TERMINATION
                 Final Time:  1.5000E+02 after    44 steps


TIMING SUMMARY
--------------
  3.158E-01 --- Total
    3.027E-03 --- Input
    2.536E-02 --- Initialization
      1.113E-03 --- mesh-MAIN
        4.888E-05 --- ghost-cells
          4.019E-05 --- cell-ghosts
            2.484E-05 --- node-neighbors
            4.810E-06 --- ghost-cells
          2.200E-07 --- link-ghosts
          7.460E-06 --- assemble-ghosts
      9.700E-06 --- VOF Initialize
      2.069E-02 --- solid mechanics
        4.944E-05 --- properties
        2.359E-04 --- precon-compute
          2.040E-04 --- residual
        1.453E-02 --- residual
        6.588E-05 --- precon-apply
      8.884E-05 --- HTSD function
        8.651E-05 --- HT function
      3.020E-06 --- ssor-setup
      1.619E-05 --- ssor-solve
    5.900E-06 --- electromagnetics
    2.834E-01 --- Main Cycle
      1.614E-05 --- Time Step
      7.046E-03 --- Diffusion Solver
        1.092E-03 --- HTSD precon compute
          1.063E-04 --- hypre-matrix-copy
          8.598E-04 --- boomer-setup
        9.743E-04 --- HTSD function
          9.445E-04 --- HT function
        3.777E-03 --- HTSD precon apply
          3.767E-03 --- HT precon apply
            3.571E-03 --- boomer-solve
      2.677E-01 --- solid mechanics
        9.764E-04 --- properties
        4.584E-03 --- precon-compute
          4.184E-03 --- residual
        2.199E-01 --- residual
        1.067E-03 --- precon-apply

                  Process virtual memory used: 6.05E+02 mB
                                   words/cell: 1239528

truchas terminated normally on 30 Jul 21 at 15:16:25
