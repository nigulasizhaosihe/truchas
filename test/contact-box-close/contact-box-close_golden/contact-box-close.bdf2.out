
TRAP step      1: T= 0.000000000000000E+00, H= 1.000000000000000E+00
  Preconditioner updated at T= 1.000000000000000E+00
    1: error= 7.009860848623321E-02
    2: error= 9.449507055736798E-04
  NLK BCE solve succeeded:   2 iterations, error= 9.449507055736798E-04
  Step accepted: no local error control

BDF2 step      2: T= 1.000000000000000E+00, H= 1.000000000000000E+00
    1: error= 4.082997620189117E-01
    2: error= 1.734271803028727E-02
    3: error= 8.016751075033610E-04
  NLK BCE solve succeeded:   3 iterations, error= 8.016751075033610E-04
  Step accepted: no local error control

BDF2 step      3: T= 2.000000000000000E+00, H= 1.000000000000000E+00
    1: error= 6.498111771626468E-02
    2: error= 1.818447688419329E-02
    3: error= 4.367885180776991E-04
  NLK BCE solve succeeded:   3 iterations, error= 4.367885180776991E-04
  Step accepted: no local error control

BDF2 step      4: T= 3.000000000000000E+00, H= 1.000000000000000E+00
    1: error= 5.170595495722797E-02
    2: error= 1.397301075361761E-02
    3: error= 1.162971055796146E-04
  NLK BCE solve succeeded:   3 iterations, error= 1.162971055796146E-04
  Step accepted: perr= 6.501335278660859E-02

BDF2 step      5: T= 4.000000000000000E+00, H= 2.679706662470576E+00
    1: error= 1.357809625246144E-01
    2: error= 1.574672123956386E-01
    3: error= 4.693090546225500E-03
  NLK BCE solve succeeded:   3 iterations, error= 4.693090546225500E-03
  Step accepted: perr= 2.920382248404181E-01

BDF2 step      6: T= 6.679706662470576E+00, H= 2.483072003962107E+00
    1: error= 1.095479510370336E-01
    2: error= 3.975459866808455E-02
    3: error= 8.177201257147774E-03
  NLK BCE solve succeeded:   3 iterations, error= 8.177201257147774E-03
  Step accepted: perr= 1.095479510370336E-01

BDF2 step      7: T= 9.162778666432683E+00, H= 4.878339193864363E+00
  Preconditioner updated at T= 1.404111786029705E+01
    1: error= 4.989635462332817E-01
    2: error= 1.149542550238924E-02
    3: error= 1.893967175543632E-04
  NLK BCE solve succeeded:   3 iterations, error= 1.893967175543632E-04
  Step accepted: perr= 4.989635462332817E-01

BDF2 step      8: T= 1.404111786029705E+01, H= 3.760676379289244E+00
    1: error= 4.989584230702546E-01
    2: error= 1.959136140786944E-02
    3: error= 2.309610736724988E-04
  NLK BCE solve succeeded:   3 iterations, error= 2.309610736724988E-04
  Step accepted: perr= 4.989584230702546E-01

BDF2 step      9: T= 1.780179423958629E+01, H= 3.827619136585049E+00
    1: error= 5.169907794561207E-01
    2: error= 3.152940886933128E-02
    3: error= 3.906027188882196E-04
  NLK BCE solve succeeded:   3 iterations, error= 3.906027188882196E-04
  Step accepted: perr= 4.987492181768114E-01

BDF2 step     10: T= 2.162941337617134E+01, H= 4.000996078548230E+00
    1: error= 5.457099392965792E-01
    2: error= 2.283042460288842E-02
    3: error= 2.198721502542036E-04
  NLK BCE solve succeeded:   3 iterations, error= 2.198721502542036E-04
  Step accepted: perr= 5.230981524615713E-01

BDF2 step     11: T= 2.563040945471957E+01, H= 3.813694237875453E+00
    1: error= 5.032313805302759E-01
    2: error= 2.877614744155004E-02
    3: error= 3.343276010426946E-04
  NLK BCE solve succeeded:   3 iterations, error= 3.343276010426946E-04
  Step accepted: perr= 4.750127898034942E-01

BDF2 step     12: T= 2.944410369259502E+01, H= 3.976276818899613E+00
    1: error= 5.297314214352292E-01
    2: error= 2.365466129692363E-02
    3: error= 2.388118707335693E-04
  NLK BCE solve succeeded:   3 iterations, error= 2.388118707335693E-04
  Step accepted: perr= 5.063161844599552E-01

BDF2 step     13: T= 3.342038051149463E+01, H= 3.737948653184169E+00
    1: error= 4.780520264778680E-01
    2: error= 3.136455734796554E-02
    3: error= 3.977590600687044E-04
  NLK BCE solve succeeded:   3 iterations, error= 3.977590600687044E-04
  Step accepted: perr= 4.519042075709256E-01

BDF2 step     14: T= 3.715832916467880E+01, H= 3.513905286329638E+00
    1: error= 4.268280165348361E-01
    2: error= 4.042620636653087E-02
    3: error= 6.381487479328232E-04
  NLK BCE solve succeeded:   3 iterations, error= 6.381487479328232E-04
  Step accepted: perr= 3.876501759230774E-01

BDF2 step     15: T= 4.067223445100844E+01, H= 3.303290522938873E+00
    1: error= 3.686694114822307E-01
    2: error= 4.565808522765630E-02
    3: error= 8.468032299086683E-04
  NLK BCE solve succeeded:   3 iterations, error= 8.468032299086683E-04
  Step accepted: perr= 3.238570357207844E-01

BDF2 step     16: T= 4.397552497394732E+01, H= 3.105299485842245E+00
    1: error= 3.150866032916652E-01
    2: error= 4.824168296198147E-02
    3: error= 1.022304310886790E-03
  NLK BCE solve succeeded:   3 iterations, error= 1.022304310886790E-03
  Step accepted: perr= 2.678660729145677E-01

BDF2 step     17: T= 4.708082445978956E+01, H= 2.919175540210439E+00
    1: error= 2.691743038485051E-01
    2: error= 4.910088668493772E-02
    3: error= 1.164595047405995E-03
  NLK BCE solve succeeded:   3 iterations, error= 1.164595047405995E-03
  Step accepted: perr= 2.212369044491544E-01

BDF2 step     18: T= 5.000000000000000E+01, H= 1.840097529930532E+00
    1: error= 1.294053190219254E-01
    2: error= 4.742672766966626E-02
    3: error= 1.804741786527322E-03
  NLK BCE solve succeeded:   3 iterations, error= 1.804741786527322E-03
  Step accepted: perr= 9.227773287488099E-02

BDF2 step     19: T= 5.184009752993053E+01, H= 1.159902470069468E+00
    1: error= 5.889104933684697E-02
    2: error= 3.270209648068065E-02
    3: error= 1.529685258523753E-03
  NLK BCE solve succeeded:   3 iterations, error= 1.529685258523753E-03
  Step accepted: perr= 2.807075644462927E-02

BDF2 step     20: T= 5.300000000000000E+01, H= 4.639609880277874E+00
    1: error= 3.840759468365548E-01
    2: error= 2.055977824549478E-02
    3: error= 2.107112368184221E-04
  NLK BCE solve succeeded:   3 iterations, error= 2.107112368184221E-04
  Step accepted: perr= 3.640434971791061E-01

BDF2 step     21: T= 5.763960988027787E+01, H= 3.624870807805571E+00
    1: error= 3.747916788376502E-01
    2: error= 2.226166030543018E-02
    3: error= 2.961045195368106E-04
  NLK BCE solve succeeded:   3 iterations, error= 2.961045195368106E-04
  Step accepted: perr= 3.747916788376502E-01

BDF2 step     22: T= 6.126448068808345E+01, H= 4.014947780460837E+00
    1: error= 5.021679218921076E-01
    2: error= 2.246064684975926E-02
    3: error= 2.338895790088676E-04
  NLK BCE solve succeeded:   3 iterations, error= 2.338895790088676E-04
  Step accepted: perr= 4.981501002468325E-01

BDF2 step     23: T= 6.527942846854428E+01, H= 4.028283894359532E+00
    1: error= 5.234147495565685E-01
    2: error= 1.866532390484880E-02
    3: error= 1.990136850133472E-04
  NLK BCE solve succeeded:   3 iterations, error= 1.990136850133472E-04
  Step accepted: perr= 5.049306252061639E-01

BDF2 step     24: T= 6.930771236290381E+01, H= 3.929333838829123E+00
    1: error= 5.158007059857327E-01
    2: error= 2.324078676929383E-02
    3: error= 2.365679078175100E-04
  NLK BCE solve succeeded:   3 iterations, error= 2.365679078175100E-04
  Step accepted: perr= 4.928027235710542E-01

BDF2 step     25: T= 7.323704620173294E+01, H= 4.003230319695305E+00
    1: error= 5.256857861758518E-01
    2: error= 2.087911746906550E-02
    3: error= 2.002541912470699E-04
  NLK BCE solve succeeded:   3 iterations, error= 2.002541912470699E-04
  Step accepted: perr= 5.050061712620444E-01

BDF2 step     26: T= 7.724027652142824E+01, H= 3.965979145935208E+00
    1: error= 5.135078942039015E-01
    2: error= 2.153639604966870E-02
    3: error= 2.106604365211337E-04
  NLK BCE solve succeeded:   3 iterations, error= 2.106604365211337E-04
  Step accepted: perr= 4.921810630444079E-01

BDF2 step     27: T= 8.120625566736345E+01, H= 4.003712574943165E+00
    1: error= 5.196223412805591E-01
    2: error= 2.026058808634458E-02
    3: error= 1.983539567298740E-04
  NLK BCE solve succeeded:   3 iterations, error= 1.983539567298740E-04
  Step accepted: perr= 4.995512019273619E-01

BDF2 step     28: T= 8.520996824230662E+01, H= 3.995281876984734E+00
    1: error= 5.175167242505699E-01
    2: error= 2.022633775122062E-02
    3: error= 1.977487423896669E-04
  NLK BCE solve succeeded:   3 iterations, error= 1.977487423896669E-04
  Step accepted: perr= 4.976450710092280E-01

BDF2 step     29: T= 8.920525011929135E+01, H= 4.002543409160225E+00
    1: error= 5.194694670961749E-01
    2: error= 2.002026763420961E-02
    3: error= 1.982538185539907E-04
  NLK BCE solve succeeded:   3 iterations, error= 1.982538185539907E-04
  Step accepted: perr= 4.996352218608934E-01

BDF2 step     30: T= 9.320779352845157E+01, H= 4.002368165267498E+00
    1: error= 5.193558973709098E-01
    2: error= 1.995067716504724E-02
    3: error= 1.981443593128454E-04
  NLK BCE solve succeeded:   3 iterations, error= 1.981443593128454E-04
  Step accepted: perr= 4.995901421551453E-01

BDF2 step     31: T= 9.721016169371907E+01, H= 4.002917514814143E+00
    1: error= 3.625661782421638E-01
    2: error= 1.345747428663963E-02
    3: error= 1.213023564557516E-04
  NLK BCE solve succeeded:   3 iterations, error= 1.213023564557516E-04
  Step accepted: perr= 3.492176690615399E-01

BDF2 step     32: T= 1.012130792085332E+02, H= 4.845256032127324E+00
    1: error= 5.697901167810538E+00
    2: error= 2.222111069705636E-01
    3: error= 3.951073298868893E-03
  NLK BCE solve succeeded:   3 iterations, error= 3.951073298868893E-03
  Step REJECTED: perr= 5.697901167810538E+00

BDF2 step     32: T= 1.012130792085332E+02, H= 2.422628016063655E+00
    1: error= 1.822333931399599E+00
    2: error= 2.392729987760300E-01
    3: error= 1.263693559050494E-02
    4: error= 4.329879185904239E-05
  NLK BCE solve succeeded:   4 iterations, error= 4.329879185904239E-05
  Step accepted: perr= 1.822333931399599E+00

BDF2 step     33: T= 1.036357072245969E+02, H= 1.456613217879394E+00
    1: error= 5.552731958213376E-01
    2: error= 2.642091467521533E-01
    3: error= 1.397847712899811E-02
    4: error= 1.481986330478446E-04
  NLK BCE solve succeeded:   4 iterations, error= 1.481986330478446E-04
  Step accepted: perr= 3.589274557924828E-01

BDF2 step     34: T= 1.050923204424763E+02, H= 2.479208027611790E+00
    1: error= 7.153329078598710E-01
    2: error= 2.030072180355545E-01
    3: error= 3.182731881054372E-03
  NLK BCE solve succeeded:   3 iterations, error= 3.182731881054372E-03
  Step accepted: perr= 5.154374299257843E-01

BDF2 step     35: T= 1.075715284700881E+02, H= 2.142463636048987E+00
    1: error= 2.005324410467803E-01
    2: error= 5.551722940063791E-02
    3: error= 7.101348200612516E-04
  NLK BCE solve succeeded:   3 iterations, error= 7.101348200612516E-04
  Step accepted: perr= 1.453570028017685E-01

BDF2 step     36: T= 1.097139921061370E+02, H= 3.956784676263254E+00
    1: error= 1.482283690994670E-01
    2: error= 1.071261117904406E-02
    3: error= 6.773695434827268E-05
  NLK BCE solve succeeded:   3 iterations, error= 6.773695434827268E-05
  Step accepted: perr= 1.375823058344851E-01

BDF2 step     37: T= 1.136707767824003E+02, H= 5.000000000000000E+00
    1: error= 1.071377893641351E-02
    2: error= 6.726302830725013E-04
  NLK BCE solve succeeded:   2 iterations, error= 6.726302830725013E-04
  Step accepted: perr= 1.108235796228496E-02

BDF2 step     38: T= 1.186707767824003E+02, H= 5.000000000000000E+00
    1: error= 2.382801325540336E-02
    2: error= 1.767222051118965E-03
  NLK BCE solve succeeded:   2 iterations, error= 1.767222051118965E-03
  Step accepted: perr= 2.517263821085003E-02

BDF2 step     39: T= 1.236707767824003E+02, H= 5.000000000000014E+00
    1: error= 1.072029884897840E-02
    2: error= 7.942534439073202E-04
  NLK BCE solve succeeded:   2 iterations, error= 7.942534439073202E-04
  Step accepted: perr= 1.132483110618465E-02

BDF2 step     40: T= 1.286707767824003E+02, H= 4.737632480605271E+00
    1: error= 2.205574933370807E-03
    2: error= 1.252257187503789E-04
  NLK BCE solve succeeded:   2 iterations, error= 1.252257187503789E-04
  Step accepted: perr= 2.291453186404578E-03

BDF2 step     41: T= 1.334084092630056E+02, H= 4.489032304257222E+00
    1: error= 9.930386888150767E-05
  NLK BCE solve succeeded:   1 iterations, error= 9.930386888150767E-05
  Step accepted: perr= 9.930386891099138E-05

BDF2 step     42: T= 1.378974415672628E+02, H= 4.253477050227048E+00
    1: error= 2.954878504621190E-04
    2: error= 6.059144807838346E-06
  NLK BCE solve succeeded:   2 iterations, error= 6.059144807838346E-06
  Step accepted: perr= 2.964674351287566E-04

BDF2 step     43: T= 1.421509186174899E+02, H= 4.030282205732959E+00
    1: error= 1.338584074456133E-04
    2: error= 3.559594331720670E-06
  NLK BCE solve succeeded:   2 iterations, error= 3.559594331720670E-06
  Step accepted: perr= 1.302988136068914E-04

BDF2 step     44: T= 1.461812008232228E+02, H= 3.818799176777190E+00
    1: error= 3.473900155909428E-05
  NLK BCE solve succeeded:   1 iterations, error= 3.473900155909428E-05
  Step accepted: perr= 3.473900091826116E-05
